test/sequential/test-inspector-port-zero-cluster.js

```diff
@@ -30,16 +30,19 @@ function serialFork() {
 if (cluster.isMaster) {
   Promise.all([serialFork(), serialFork(), serialFork()])
     .then(common.mustCall((ports) => {
-      ports.push(process.debugPort);
-      ports.sort();
+      ports.splice(0, 0, process.debugPort);
       // 4 = [master, worker1, worker2, worker3].length()
       assert.strictEqual(ports.length, 4);
       assert(ports.every((port) => port > 0));
       assert(ports.every((port) => port < 65536));
-      // Ports should be consecutive.
-      assert.strictEqual(ports[0] + 1, ports[1]);
-      assert.strictEqual(ports[1] + 1, ports[2]);
-      assert.strictEqual(ports[2] + 1, ports[3]);
+      ports.forEach((port, i) => {
```

Comment:
I think I'd prefer to keep the existing assertion style instead of using a `forEach()`. It's much more readable IMO.